### PR TITLE
Donations fixes

### PR DESCRIPTION
**Pull request by Av1sa**

### CODE REVIEW

File: components/FormValidator.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    inputList.forEach((input) => {
+      this._isValid(input);
+      input.addEventListener("input", () => {
+        this._isValid(input);
+        this._toggleButtonState(inputList, submitBtn);
+      });
+    });
+  }
+
+  //Check if any field from list is invalid
```
Comment (julieg18) [inline components/FormValidator.js:62]: General comment, we should probably delete comments.

File: components/FormValidator.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  _setEventListeners() {
+    const inputList = Array.from(
+      this._element.querySelectorAll(this._inputSelector)
+    );
+    const submitBtn = this._element.querySelector(this._submitButtonSelector);
+    this._toggleButtonState(inputList, submitBtn);
+
+    inputList.forEach((input) => {
+      this._isValid(input);
+      input.addEventListener("input", () => {
```
Comment (julieg18) [inline components/FormValidator.js:55]: General comment, you should make all your strings single quote to improve consistency.

File: components/FormValidator.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  //Check all fields in form
+  _setEventListeners() {
+    const inputList = Array.from(
+      this._element.querySelectorAll(this._inputSelector)
+    );
+    const submitBtn = this._element.querySelector(this._submitButtonSelector);
+    this._toggleButtonState(inputList, submitBtn);
+
+    inputList.forEach((input) => {
+      this._isValid(input);
```
Comment (julieg18) [inline components/FormValidator.js:54]: Delete this line of code and the error messages won't show until you start typing.

File: components/FormValidator.js (hunk 1 above)
Comment (Av1sa) [reply components/FormValidator.js:62]: Why? All coders leave comments in their code for better understanding what's going on in future

File: components/FormValidator.js (hunk 2 above)
Comment (Av1sa) [reply components/FormValidator.js:55]: All my strings are consistent and double quote - that's the way my save&format plugin works

Comment (julieg18) [reply components/FormValidator.js:55]: Can you change it to single? We originally had single quote as our string type. We don't wanr some files made with single quote and other files with double quotes.

File: components/FormValidator.js (hunk 1 above)
Comment (julieg18) [reply components/FormValidator.js:62]: The function names appear to tell what's happening in each function, but if you want to keep them, that's perfectly fine😀

File: components/FormValidator.js (hunk 2 above)
Comment (Av1sa) [reply components/FormValidator.js:55]: We already have double quotes in index.html, by the way :)

**Pull request merged**